src/hotspot/cpu/riscv/vm_version_riscv.cpp

```diff
@@ -249,13 +249,6 @@ void VM_Version::initialize() {
     FLAG_SET_DEFAULT(UseBlockZeroing, false);
   }
 
-  // This machine allows unaligned memory accesses
-  if (FLAG_IS_DEFAULT(UseUnalignedAccesses)) {
-    FLAG_SET_DEFAULT(UseUnalignedAccesses,
-      unaligned_access.value() != MISALIGNED_UNKNOWN &&
```

Comment:
is MISALIGNED_FAST the right value to check against (which is what the other version uses)?

Reply:
Yes, as it's done already at https://github.com/rivosinc/jdk/pull/133/files#diff-7b173d6e5834de13749c8333192fef5a874628a67b90a5d8d06235d507542ac4R227-R230